contrib/gcppubsub/config/rbac_role_binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: gcppubsub-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: gcppubsub-controller-role
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
Should we also create a unique service account?

Reply:
Good point, done.

I also added the necessary startup secret (as an empty string) to make initial install start without waiting for the secret to exist.